download_and_create_reference_datasets/v03/write_cached_reference_dataset_query_ht.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+
+import hail as hl
+
+from hail_scripts.reference_data.config import CONFIG
+
+from v03_pipeline.lib.misc.io import write
+from v03_pipeline.lib.model import CachedReferenceDatasetQuery, Env, ReferenceGenome
+from v03_pipeline.lib.paths import cached_reference_dataset_query_path
+
+
+def run(
+    env: Env,
+    reference_genome: ReferenceGenome,
+    query: CachedReferenceDatasetQuery,
+):
+    config = CONFIG[query.dataset][reference_genome.v02_value]
+    ht = (
```

Comment:
Using the raw reference dataset rather than the combined version because the required fields may be different.